### DISCUSSION TITLE
Importing Custom Ruleset From Tfsec to Scan CDK Cloudformation Files.

### DISCUSSION THREAD

**Discussion by mxmclark** [aquasecurity/trivy#3931] [Q&A] [answered]
Hello,
I am pretty new to Trivy and I have been tasked with importing a custom rule set that is already made called tfsec-rules into trivy in order to scan for cdk cloud formation files. How do i go about importing the files? The rules are already written in Rego. What folder would i need to import these rules to, in order to use them to scan these CDK files using trivy config. 
Thank you feel free to reach out.

**Comment by simar7** [comment, marked as answer]
hi @ma031932 - Trivy supports evaluating custom policies. You can read more on that here https://aquasecurity.github.io/trivy/v0.41/docs/scanner/misconfiguration/custom/schema/#custom-policies-with-custom-schemas

**Reply by AnaisUrlichs** [reply]
You can put those files, custom Rego policies and the custom schema for your resources, into any folder you would like, you just have to reference the policies with the `--policy` flag 
Here is an example I used for a presentation for scanning ArgoCD custom resources: https://github.com/Cloud-Native-Security/gitops-the-magickey
